pkg/logging/logError.go

```diff
@@ -0,0 +1,7 @@
+package logging
+
+func Error(err error, keypairs ...interface{}) {
+	if err != nil {
+		Context().Logger(DefineSubRealm("error")).Error(err.Error(), keypairs...)
```

Comment:
The context comes from the library / CLI. The problem with doing this is that it completely ignores what the context setup is. It could be the CLI which logs to stdout or error or the Library where the logger will log to a configured medium that is not represented here.

Either way, this will not work as the context is not passed into these situations so it will log to an arbitrary location. Worst case, it displays a warning saying something like this

```
[controller-runtime] log.SetLogger(...) was never called; logs will not be displayed.

```

Reply:
argh... and a dump and simple: `println()` ?

Reply:
That still could produce an unexpected output to the wrong logging medium. :(